### PR TITLE
MGMT-1263 Allow the overriding of the rhcos image

### PR DESCRIPTION
**Pull request by asalkeld**

### CODE REVIEW

**Review by avishayt — CHANGES_REQUESTED** [review]
Looks like this is MGMT-2262, not 1262.  Can you please explain why you need to override the image?

**Comment by asalkeld** [comment]
> Looks like this is MGMT-2262, not 1262. Can you please explain why you need to override the image?

Sorry @avishayt typo.. it's 1263 - i'll fix the headline

**Review by avishayt — APPROVED** [review]

File: internal/common/error_utils.go [hunk 1]
```diff
@@ -22,7 +23,7 @@ func GenerateError(id int32, err error) *models.Error {
 
 func GenerateInternalFromError(err error) *models.Error {
 	return &models.Error{
-		Code:   swag.String(string(http.StatusInternalServerError)),
+		Code:   swag.String(fmt.Sprint(http.StatusInternalServerError)),
```
Comment (eranco74) [inline internal/common/error_utils.go:26]:
```suggestion
		Code:   swag.String(fmt.Sprintf(http.StatusInternalServerError)),
```

**Review by eranco74 — COMMENTED** [review]
Looks good, just a typo

**Comment by eranco74** [comment]
@asalkeld 
Your commit message is missing either a JIRA issue ('JIRA-1111'), a GitHub issue ('#39').

[2020-11-19T02:03:57.255Z] You can also ignore the ticket checking with 'NO-ISSUE'.

**Review by avishayt — APPROVED** [review]

**Pull request merged**